packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -524,6 +533,85 @@ class Viewport implements IViewport {
     return intersections;
   }
 
+  /**
+   * Sets the camera to an initial bounds. If
+   * resetPan and resetZoom are true it places the focal point at the center of
+   * the volume (or slice); otherwise, only the camera zoom and camera Pan or Zoom
+   * is reset for the current view.
+   * @param displayArea - The display area of interest.
+   * @param storeAsInitialCamera - If true, reset camera is stored as the initial camera (to allow differences to
+   *   be detected for pan/zoom values)
+   * @param suppressEvents - If true, don't fire displayArea event.
+   */
+  public setDisplayArea(
+    displayArea: DisplayArea,
+    storeAsInitialCamera = false,
+    suppressEvents = false
+  ): void {
+    this.setCamera(this.initialDisplayAreaCamera, false); // ensure displayArea is absolute
+
+    const {
+      imageArea = { areaX: 1, areaY: 1 },
+      imageFocalPoint = { focalX: 0.5, focalY: 0.5 },
+    } = displayArea;
+    // setPan functions requires offsetviewer to be initialized, which does not seem to be the case at constructor time
```

Comment:
The most ideal is if we can start this initial display logic when the vtkoffset renderer is able to perform its canvas to world calculations. For some reason these pop up as NaN during constructor time